wagtailio/areweheadlessyet/signal_handlers.py

```diff
@@ -0,0 +1,39 @@
+from threading import Thread
+
+import requests
+from django.conf import settings
+from wagtail.core.signals import page_published, page_unpublished
+
+from .models import AreWeHeadlessYetHomePage, AreWeHeadlessYetTopicPage
+
+
+def postpone(function):
+    """
+    Cheap aysnc, see https://stackoverflow.com/a/28913218
+    """
+
+    def decorator(*args, **kwargs):
+        t = Thread(target=function, args=args, kwargs=kwargs)
+        t.daemon = True
+        t.start()
+
+    return decorator
+
+
+@postpone
+def deploy(sender, **kwargs):
+    """Triggers a build on Vercel."""
+
+    response = requests.post(settings.VERCEL_DEPLOY_HOOK_URL)
```

Comment:
We should add a timeout here, in case there is a network issue (requests will wait forever by default)